lib/galaxy/model/mapping.py

```diff
@@ -557,6 +557,26 @@
         ForeignKey("history_dataset_collection_association.id"), index=True),
     Column("name", Unicode(255)))
 
+model.ImplicitCollectionJobs.table = Table(
+    "implicit_collection_jobs", metadata,
+    Column("id", Integer, primary_key=True),
+    Column("populated_state", TrimmedString(64), default='new', nullable=False),
+)
+
+# model.ImplicitCollectionJobsHistoryDatasetCollectionAssociation.table = Table(
```

Comment:
are we dropping this whole table?

Reply:
If not (and the migration seems to not to) we might want to track this (and others) dead/unused tables.

Reply:
More left overs from a messy development process - that was never added and I'll just drop the commented code. Thanks for the catch.